packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx

```diff
@@ -39,7 +39,7 @@ export const CheckboxPrimitive: FC<Props> = ({
 
   return (
     <Fragment>
```

Comment:
That `<Fragment>` wrapper is not needed, not sure why it was there.

Reply:
Will be removed